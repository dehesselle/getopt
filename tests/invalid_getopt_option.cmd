#!/bin/sh
"$1" -b
