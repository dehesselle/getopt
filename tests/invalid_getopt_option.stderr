$0: invalid option -- 'b'
Try `getopt --help' for more information.
